superset-frontend/src/views/App.tsx

```diff
@@ -43,26 +49,39 @@ const bootstrap = JSON.parse(container?.getAttribute('data-bootstrap') ?? '{}');
 const user = { ...bootstrap.user };
 const menu = { ...bootstrap.common.menu_data };
 const common = { ...bootstrap.common };
+let lastLocation: string;
```

Comment:
@ktmud lastLocation is string type. I think the name is really misleading, i fixed as Erik suggested.